backend/src/main/java/com/google/sps/servlets/AddParticipantServlet.java

```diff
@@ -74,19 +66,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
     JSONObject formDetails = obj.getJSONObject(REQUEST_FORM_DETAILS);
 
-    // Retrieve the timeAvailableUntil input and convert to a UTC ZonedDateTime
-    long timeAvailableUntil = formDetails.getLong(REQUEST_TIME_AVAILABLE_UNTIL);
-    ZoneId zoneId = ZoneId.of("UTC");
-    ZonedDateTime startTimeAvailable = ZonedDateTime.now(Clock.systemUTC());
-    ZonedDateTime endTimeAvailable =
-        ZonedDateTime.ofInstant(Instant.ofEpochMilli(timeAvailableUntil), zoneId);
+    // Retrieve the timeAvailableUntil input and convert to a UTC milliseconds
```

Comment:
`// Retrieve timeAvailableUntil input and calculate startTimeAvailable in terms of milliseconds` ?